var/spack/repos/builtin/packages/phist/package.py

```diff
@@ -235,6 +235,13 @@ def patch(self):
         test.filter("1 2 3 12", "1 2 3")
         test.filter("12/", "6/")
         test.filter("TEST_DRIVERS_NUM_THREADS 6", "TEST_DRIVERS_NUM_THREADS 3")
+        # Avoid finding external modules like:
+        #    /opt/rocm/llvm/include/iso_fortran_env.mod
+        filter_file(
+            "use iso_fortran_env",
+            "use, intrinsic :: iso_fortran_env",
+            "drivers/matfuncs/matpde3d.F90",
+        )
```

Comment:
@v-dobrev, @jthies  perhaps this fix should be limited to current release [assuming it can be added to upstream - for future releases]?

Reply:
I agree -- just note that the problematic code is present in older versions too. Once there is a fix for this in the Phist sources, the patching code can be made version dependent. I'm not sure if there is a minimum version to which this should be applied to.

Reply:
> Once there is a fix for this in the Phist sources, the patching code can be made version dependent.

Ok - so we can merge in the current change for now - assuming @jthies can approve